packages/flutter/test/material/tab_bar_theme_test.dart

```diff
@@ -1791,4 +1791,58 @@ void main() {
     targetRect = const Rect.fromLTRB(275.0, 0.0, 325.0, 48.0);
     expectIndicatorAttrs(tabBarBox, rect: rect, targetRect: targetRect);
   });
+
+  testWidgets('splashBorderRadius is passed from TabBarThemeData', (WidgetTester tester) async {
```

Comment:
optional wording suggestion:

```suggestion
  testWidgets('TabBar inherits splashBorderRadius from theme', (WidgetTester tester) async {
```